source/config.js

```diff
@@ -212,3 +214,22 @@ Object.defineProperty(Error.prototype, 'toJSON', {
   },
   configurable: true
 });
+
+try {
+  module.exports.gitVersion = /.*?(\d+[.]\d+[.]\d+).*/.exec(child_process.execSync('git --version').toString())[1];
```

Comment:
I think the reason these were async before was to shorten the startup time. Not sure if it actually is a big difference though, but might be good to look out for (i.e. if it's significantly slower maybe we should think about keeping the async version)